docs/source/tutorial/dynamodb.rst

```diff
@@ -0,0 +1,293 @@
+
+
+Using an Existing Table
+-----------------------
+It is also possible to create a :py:class:`DynamoDB.Table` resource from
+an existing table::
+
+    import boto3
+
+    # Get the service resource.
+    dynamodb = boto3.resource('dynamodb')
+
+    # Instantiate a table resource object without actually
+    # creating a DynamoDB table. Note that the attributes of this table
+    # are lazy-loaded: a request is not made nor are the attribute
+    # values populated until the attributes
+    # on the table resource are accessed or its load() method is called.
+    table = dynamodb.Table('users')
+
+    # Print out some data about the table.
+    # This will cause a request to be made to DynamoDB and its attribute
+    # values will be set based on the response.
+    print(table.creation_date_time)
+
+
+Creating a New Item
+-------------------
+
+Once you have a :py:class:`DynamoDB.Table` resource you can add new items
+to the table using :py:meth:`DynamoDB.Table.put_item`::
+
+    table.put_item(
+       Item={
+            'username': 'janedoe',
+            'first_name': 'Jane',
+            'last_name': 'Doe',
+            'age': 25,
+            'account_type': 'standard_user',
+        }
+    )
+
+For all of the valid types that can be used for an item, refer to
+:ref:`ref_valid_dynamodb_types`.
+
+
+Getting an Item
+---------------
+You can then retrieve the object using :py:meth:`DynamoDB.Table.get_item`::
+
+    response = table.get_item(
+        Key={
+            'username': 'janedoe',
+            'last_name': 'Doe'
+        }
+    )
+    item = response['Item']
+    print(item)
+
+
+Updating Item
+-------------
+
+Using the retrieved item, you can update attributes of the item in the table::
+
+    item['age'] = 26
+    table.put_item(Item=item)
+
+Then if you retrieve the item again, it will be updated appropriately::
+
+    response = table.get_item(
+        Key={
+            'username': 'janedoe',
+            'last_name': 'Doe'
+        }
+    )
+    item = response['Item']
+    print(item)
+
+
+Deleting Item
+-------------
+You can also delete the item using :py:meth:`DynamoDB.Table.delete_item`::
+    
+    table.delete_item(
+        Key={
+            'username': 'janedoe',
+            'last_name': 'Doe'
+        }
+    )
+
+
+Batch Writing
+-------------
+If you are loading a lot of data at a time, you can make use of
+:py:meth:`DyanmoDB.Table.batch_writer` so you can both speed up the process and
+reduce the number of write requests made to the service.
+
+This method returns a handle to a batch writer object that will automatically
+handle buffering and sending items in batches.  In addition, the
+batch writer will also automatically handle any unprocessed items and
+resend them as needed.  All you need to do is call ``put_item`` for any
+items you want to add, and ``delete_item`` for any items you want to delete::
+
+    with table.batch_writer() as batch:
+        batch.put_item(
```

Comment:
Yeah I could add something like that. I still would want to keep the codeblock for the four items that I have because it is needed for the query and scanning section. I could add a codeblock underneath of it adding a bunch of more users or something. I also link to the method's reference docs and those have an example already in it that uses a large for loop: http://boto3.readthedocs.org/en/latest/reference/services/dynamodb.html#DynamoDB.Table.batch_writer

Let me know what you think. I would be fine adding that codeblock underneath the original example.